src/codewhisperer/service/classifierTrigger.ts

```diff
@@ -397,6 +817,78 @@ export class ClassifierTrigger {
         return shouldTrigger
     }
 
+    private calculateResultForExperimentGroup(
```

Comment:
I assume this is generated, and tested on some backend system somewhere. Else it might be worth covering with a test.

Reply:
yes we are doing e2e tests to make sure the calculations are correct